spec/rails_helper.rb

```diff
@@ -5,7 +5,6 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
-require 'webmock/rspec'
```

Comment:
@dhalverson  I tried removing this from rails helper in an attempt to troubleshoot our travis-ci failures. In the lesson plan is says to include in spec helper and I think having it here may block the API calls? I dunno but we can add this back in if it doesn't make a difference.

Reply:
This seems to be ok, if it causes issues it's an easy reversal.